config/environments/development.rb

```diff
@@ -25,8 +25,8 @@
     config.cache_store = :null_store
   end
 
-  config.active_storage.service = if ENV['LAGO_USE_AWS_S3'].present?
-    if ENV['LAGO_AWS_S3_ENDPOINT'].present?
+  config.active_storage.service = if ENV['LAGO_USE_AWS_S3'].present? && ENV['LAGO_USE_AWS_S3'] == 'true'
+    if ENV['LAGO_AWS_S3_ENDPOINT'].present? && !ENV['LAGO_AWS_S3_ENDPOINT'].empty?
```

Comment:
Seems to be redondant here, if it's present, it's not empty, right? 😅 

```
"".present? => false
[].present? => false
nil.present? => false
{}.present? => false
```

Reply:
2 test is better than one (but yes :joy:)

Reply:
btw this thing start to annoy me a lot, I will work on a Lago's config file soon!